src/console.ts

```diff
@@ -52,4 +53,8 @@ export class ConsoleMessage {
   location(): ConsoleMessageLocation {
     return this._location;
   }
+
+  [util.inspect.custom]() {
+    return this.text();
```

Comment:
Should we add more details? E.g. `[warn] .... @foo.js:15`.

Reply:
Hm. That's not what the regular console.log does in Node though, maybe that should be opt-in?